src/components/SingleReview.jsx

```diff
@@ -0,0 +1,14 @@
+import {useState} from 'react'
+import SingleReviewCard from "./SingleReviewCard"
+
+const SingleReview = () => {
```

Comment:
Nothing wrong with this per se but having a SingleReview component that just contains a SingleReviewCard is a bit redundant here